INSTALL.md

```diff
@@ -55,3 +55,13 @@ python setup.py build develop
 # or if you are on macOS
 # MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py build develop
 ```
+
+### Option 2: Docker Image (Requires CUDA)
```

Comment:
Could you also add a note saying that it doesn't work with OSX?

Reply:
sorry, didn't see this. why doesn't it work in OSX btw?